app/Config/CURLRequest.php

```diff
@@ -6,6 +6,22 @@
 
 class CURLRequest extends BaseConfig
 {
+    /**
+     * --------------------------------------------------------------------------
+     * CURLRequest Share Connection
+     * --------------------------------------------------------------------------
+     *
+     * Whether share connection between requests.
```

Comment:
This is what I mean:
```suggestion
     * --------------------------------------------------------------------------
     * CURLRequest Share Connection Options
     * --------------------------------------------------------------------------
     *
     * Share connection options between requests.
```